src/App.css

```diff
@@ -3,12 +3,14 @@
 
 :root {
   --half-canva-blue:rgb(0, 196, 204, 50%);
-  --light-white: rgb(220, 220, 220);
+  --light-white: rgb(100, 100, 100);
```

Comment:
hmm why are the changes from #15 in this PR too?

Reply:
I rebranched from `mp3-files` so that I can base this PR from `mp3-files`?